main.py

```diff
@@ -40,9 +40,6 @@ async def unload(ctx, extension):
 #     role = server.get_role(992232900982476800)
 #     await member.add_roles(role)
 
-# @client.command()
```

Comment:
Are we just removing this function completely? Should we keep this and just move it to a different cog rather than `main.py`?

Reply:
Removing it because it has been moved to another cog